config/config-package.py

```diff
@@ -356,11 +356,12 @@ def tox(self):
                 'test_environment_variables',
                 'extra_lines',
                 'use_pytest_plone',
+                'package_name'
             )
         )
         use_mxdev = options.get("use_mxdev", False)
         options.update(self._test_cfg())
-        options['package_name'] = self.path.name
+        package_name = options.get("package_name", self.path.name)
```

Comment:
I did not try it, but I think you should update the `options`.  Otherwise unless the package_name is given explicitly the `package_name` key will be missing, and this likely gives an error.

```suggestion
        if not options.get("package_name"):
            options["package_name"] = self.path.name
        
```